### DISCUSSION TITLE
怎么开启夜间模式？ 像 dumi1 那样的第三方链接的 icon

### DISCUSSION THREAD

**Discussion by 2531800823** [umijs/dumi#1375] [General]
<img width="375" alt="WeChat3b2508520ae7799db9d0bbd48cbd492a" src="https://user-images.githubusercontent.com/75556006/207551799-ad5c7e2d-896d-49de-a7f3-9460100bc08e.png">

**Comment by PeachScript** [comment]
暂时还不支持暗黑模式，预计这个月内会支持

**Reply by 2531800823** [reply]
像 dumi1 那样的第三方链接的 icon，还有底部！
<img width="1573" alt="image" src="https://user-images.githubusercontent.com/75556006/207565234-0f18d6ee-0a69-4c88-8991-8b237dc89e03.png">

**Comment by PeachScript** [comment]
暗黑模式已支持，讨论关闭
